pkg/api/const.go

```diff
@@ -457,7 +457,7 @@ const (
 	// DefaultKubernetesDNSServiceIPv6 specifies the IPv6 address that kube-dns listens on by default. must by in the default Service CIDR range.
 	DefaultKubernetesDNSServiceIPv6 = "fd00::10"
 	// DefaultMobyVersion specifies the default Azure build version of Moby to install.
-	DefaultMobyVersion = "3.0.10"
+	DefaultMobyVersion = "19.03.11"
```

Comment:
19.03.12 is latest.

Can't believe we missed this one.

Reply:
seems on github it says latest release is 19.03.11: https://github.com/Azure/moby/releases
also 11 was released on Jun.26 but 12 was released on Jun.25 which looks weird

Reply:
That's just tag date vs when the release was created in GH.

Reply:
One is automatic when a tag is created, the other is manual after the fact.

Reply:
OK sounds good. I'll update to .12